documentation/documentation.html

```diff
@@ -291,9 +291,199 @@ <h4 class="card-title">Sample Card</h4>
                         sandbox="allow-scripts allow-same-origin">
                     </iframe>
                 </div>
+            </div>
+
+
+            <!---------------------button---------------------->
+            <div class="button">
+                <div class="comp-intro">
+                    <h1 class="main-h1">Buttons</h1>
+                    <p class="main-p weight-mid">Use PixelUI’s custom button styles for actions in forms, dialogs, and
+                        more with support for multiple sizes, states, and more.</p>
+                </div>
+                <h2 class="main-h2">
+                    Example of Primary Buttons
+                </h2>
+                <p class=doc-p>
+                    Primary buttons are of various types - Primary button, Accent button, Secondary button.
+                    <br>
+                    You can add various colors according to your color scheme using classes like <span
+                        class=prime-color-text>btn-primary, btn-secondary, btn-accent.</span>
+                    <br>
+                    <br>
+                    <code class="main-code">
+                        btn:
+                    </code>
+                    To add padding and font size
+                    <br>
```

Comment:
Don't use `br` element to add the line break. Instead, use flexbox to arrange the elements.